zerver/models.py

```diff
@@ -3024,7 +3024,6 @@ class AbstractMessage(models.Model):
 
     # A JSON-encoded list of objects describing any past edits to this
     # message, oldest first.
```

Comment:
This comment becomes out of date here?  I mean, I guess a JSONField is _kinda_ "JSON-encoded" but that seems like it's not being the most helpful. :)

Reply:
Reworded it in the commit that removes the legacy field.